src/main/java/core/basesyntax/Drawable.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface Drawable {
+    public abstract String draw();
```

Comment:
To 'draw' means to print out all information about a figure using System.out.println()
You should `print` inside the method `draw`
```suggestion
    public void draw();
```